README.md

```diff
@@ -45,7 +45,11 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * Trim the audio if required
 
-        $ sox cleaned.wav final.wav trim 00:02 =21:20 
+        $ sox cleaned.wav final.wav trim 00:02 =21:20
+
+* Trim the video if required
+
+        $ ffmpeg -i <input_video_name> -ss <start_time_in_hh:mm:sec> -to <end_time_in_hh:mm:sec> <output_video_name>
```

Comment:
I think you'll have to move `-ss` before `-i`

FWIW, prepending `ss` seeks to keyframe (faster) vs appending `ss` decodes and discards until that time is reached (slower)

checkout : https://trac.ffmpeg.org/wiki/Seeking